crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -116,79 +116,43 @@ bitflags::bitflags! {
 
 pub fn extract_meshes(
     mut commands: Commands,
-    mut previous_caster_len: Local<usize>,
-    mut previous_not_caster_len: Local<usize>,
-    caster_query: Query<
-        (
-            Entity,
-            &ComputedVisibility,
-            &GlobalTransform,
-            &Handle<Mesh>,
-            Option<&NotShadowReceiver>,
-        ),
-        Without<NotShadowCaster>,
-    >,
-    not_caster_query: Query<
-        (
-            Entity,
-            &ComputedVisibility,
-            &GlobalTransform,
-            &Handle<Mesh>,
-            Option<&NotShadowReceiver>,
-        ),
-        With<NotShadowCaster>,
-    >,
+    mut prev_len_shadow_caster: Local<usize>,
+    mut prev_len_not_shadow_caster: Local<usize>,
```

Comment:
Yeah. Those names are terrible. I'll opt for `prev_{not_,}caster_commands_len` since `{not_,}caster_commands` is the name of the buffer we use for the bundles to spawn.